src/main/java/by/itninjas/reposiroty/impl/EmployeeRepositoryImpl.java

```diff
@@ -1,48 +1,48 @@
 package by.itninjas.reposiroty.impl;
 
-import by.itninjas.domain.entity.User;
-import by.itninjas.reposiroty.UserRepository;
+import by.itninjas.domain.entity.Employee;
+import by.itninjas.reposiroty.EmployeeRepository;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;
 import org.springframework.stereotype.Repository;
 
 @Repository
-public class UserRepositoryImpl implements UserRepository {
+public class EmployeeRepositoryImpl implements EmployeeRepository {
 
-    private final Map<Integer, User> STORAGE = new ConcurrentHashMap<>();
+    private final Map<Integer, Employee> STORAGE = new ConcurrentHashMap<>();
```

Comment:
In some cases you can use some beans of any type. So for STORAGE better use static field because all instances can have access to one storage. Make it static